<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="calibration_start">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start calibration</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Reward</string>
    </property>
    <widget class="QLineEdit" name="serial_path">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>The address of the serial port used to send the reward signal</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid lightgray;
border-radius: 5px</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial port: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>130</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Reward duration [s] :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="calibration_reward_duration">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>The amount of reward to give during calibration</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid lightgray;
border-radius: 5px</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Screen</string>
    </property>
    <widget class="QLineEdit" name="screen_size">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Diagonal of the monitor</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid lightgray;
border-radius: 5px</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pixel height :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="screen_width">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Width of the screen in pixels</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid lightgray;
border-radius: 5px</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pixel width :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance [cm]  :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Physical size [inches] :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="screen_distance">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Distance from the monitor to the subject's eye</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid lightgray;
border-radius: 5px</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="screen_height">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Height of the monitor in pixels</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid lightgray;
border-radius: 5px</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Calibration</string>
    </property>
    <widget class="QCheckBox" name="manual_calibration">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Whether to use manual calibration</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Manual calibration: </string>
     </property>
    </widget>
    <widget class="QComboBox" name="calibration_type">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The type of calibration to use</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>130</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibration stimulus :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>130</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibration color :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Target size [deg] :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="calibration_target_size">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>The size of the calibration target, in degress of visual angle</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid lightgray;
border-radius: 5px</string>
     </property>
    </widget>
    <widget class="QComboBox" name="calibration_stimulus">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The type of calibration stimulus to use</string>
     </property>
    </widget>
    <widget class="QComboBox" name="calibration_target_color">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The color of the calibration stimulus</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>130</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibration type :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
